### ISSUE TITLE
question about thermocouple inputs

### ISSUE THREAD

**Issue by rocketbob** [garyStofer/AIR_TFT_12ADC#1]
Are your K-type thermocouples connected directly to the LTC2495 or is there any signal conditioning or isolation in-between?

**Comment by garyStofer** [comment]
Yes the gain in the LTC is set so that the TC probes go directly on it’s inputs. 

My application is on an aircraft motor and I found that all but one of my probes had a short to chassis at the tip, inherently in the way they are made. This together with current from the alternator flowing through the chassis caused ground loops in the returns of the various probes, so I decided to use differential input pairs on the LTC, therefore reducing the number of probes down to 8 per device.  Using weak pull ups on the +inputs and weak pull downs on the – inputs I bias the input common mode to be in the middle of the devices input range.  This also serves as a indication for an open TC  because the signal rails in that case.  Off the top of my head I think I used 330 or  470K resistors. Also, this requires that the power supply for the device is ground free,  a DC to DC converter followed by a linear regulator is needed ( or a separate battery).

Maybe your application is not hampered by the above application issue and you can use the single input/ shared common mode the deice offers.

Cheers 

Gary

  
From: rocketbob 
Sent: Monday, February 12, 2018 6:37 AM
To: garyStofer/AIR_TFT_12ADC 
Cc: Subscribed 
Subject: [garyStofer/AIR_TFT_12ADC] question about thermocouple inputs (#1)

Are your K-type thermocouples connected directly to the LTC2495 or is there any signal conditioning or isolation in-between?

**Comment by rocketbob** [comment]
Thanks for the info Gary.

My application is exactly the same thing, EGT/CHT's on a Lycoming (RV-6).  Probes are grounded K-type.  I was previously using max6675's and as I recall those devices use a similar pull-up/down scheme.

Thanks,
Bob

**Comment by garyStofer** [comment]
Yes,  --  I have an RV8  with a Franklin 220 & CS prop , but this went into my Stinson with the same engine. 

if you get my PCB fro the board from OSH note that I have not completely modified it for the pull resistors yet, There where 2 cuts and jumpers needed and wider screw terminals with the wider spacing to skip over the  GND pins.

**Comment by rocketbob** [comment]
Love the Franklin 220.  I used to maintain a Stinson with one.  Have a neighbor that has a 108 with the 165.  I'm also building a F1 with a 540.  A friend of mine who passed away a few years ago was the originator of the Super 8, and he used to let me fly his -8 with the 540.  Another friend of mine inherited the engine mount jig.  You're going to miss that 220 in the -8!

Is the PCB available on Oshpark?  If so I don't see it.

**Comment by rocketbob** [comment]
Yes I definitely will take the board.  Just shoot me your email and I will
paypal you.  I just got some sample LTC2495's yesterday and was
contemplating how to prototype.  I have a smt soldering microscope, hot
air, etc.

Thanks,
Bob

**Comment by garyStofer** [comment]
Excellent --  PP me at garystofer@hotmail.com  -

I figure a flat pack in the US comes to about $4, so send about $13 and your address so I can get it out today ( I’m out of town for the rest of the week).
If you are interested in the code for the project I can check it in to Github. I think there is something there already, but I probably have some revised version sitting on the dev machine still. It’s using an ATMEL 328P aka Arduino and I’m using a controller board I made for a 2 1/4”round instrument hole with a one of those 240x180 graphic LCDs do display bars.  It’s just big enough to display 6 bars at a time and conveniently read it without having to take the magnifying glass.   The main PCB is also on OSH under   Air_LCDuino, has a rotary knob and  the same footprint as the ADC.  I don’t have any extras of those however.   Note that I didn’t use the SD card slot on the ADC board but instead the bigger one that’s on the LCD board. 

I mounted the ADC board right on the firewall ( cold side) and then only have to run a 4 wire cable to the instrument head. Slowed I2C speed to  20Khz ( I think) .

Gary



 
Cc: Gary Stofer ; Comment 
Subject: Re: [garyStofer/AIR_TFT_12ADC] question about thermocouple inputs (#1)

Yes I definitely will take the board. Just shoot me your email and I will
paypal you. I just got some sample LTC2495's yesterday and was
contemplating how to prototype. I have a smt soldering microscope, hot
air, etc.

Thanks,
Bob

**Comment by garyStofer** [comment]
here is the permalink for the “mother” board https://oshpark.com/shared_projects/kT6TDwLU

**Comment by garyStofer** [comment]
got the money,  will go out tomorrow via USPS – 

Keep me posted of your progress .

Thanks 

Gary
